src/base-middleware.ts

```diff
@@ -47,7 +47,7 @@ export function setupBaseMiddleware(ctx: AppContext, expressApp: Express) {
             req.ctx.addLoggerExtra('req', {
                 id: req.id,
                 method: req.method,
-                url: ctx.utils.redactSensitiveQueryParams(req.url),
+                url: ctx.utils.redactSensitiveQueryParams(req.path),
```

Comment:
If i remember correctly `path` and `url` can be insufficient.

https://stackoverflow.com/revisions/69647669/5